src/polyfem/io/OutData.cpp

```diff
@@ -92,9 +92,36 @@ namespace polyfem::io
 
 				vals.compute(e, state.mesh->is_volume(), points, bs, gbs);
 
+				// for (int n = 0; n < normals.rows(); ++n)
+				// {
+				// 	normals.row(n) = normals.row(n) * vals.jac_it[n];
+				// 	normals.row(n).normalize();
+				// }
+
 				for (int n = 0; n < normals.rows(); ++n)
 				{
-					normals.row(n) = normals.row(n) * vals.jac_it[n];
+					trafo = vals.jac_it[n].inverse();
```

Comment:
what this not already fixed?

Reply:
It is fixed for the output but not for computing the continuous traction forces. In any case, the continuous traction forces are too inaccurate to use.